src/features/people/PersonPage/index.js

```diff
@@ -4,15 +4,20 @@ import { nanoid } from 'nanoid'
 import { Tile } from '../../../common/components/Tiles/Tile'
 import { Loader } from '../../../common/components/Loader'
 import { Container } from '../../../common/components/Container'
-import { selectPersonPage, fetchPersonPage, selectError, selectLoading } from './personPageSlice'
+import { selectPersonPage, fetchPersonPage, selectPersonCast } from './personPageSlice'
 import { Title } from '../../../common/components/Title'
 import { useEffect } from "react"
 import { ErrorMessage } from '../../../common/components/ErrorMessage'
+import {PersonCast} from './PersonCast'
+import { PersonInfo } from "./PersonInfo"
```

Comment:
you should delete them when you didn't use them It causes that everything collapse